.github/workflows/python_tests.yml

```
@@ -0,0 +1,33 @@
+name: python_tests
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  python_tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [2.7, 3.8]  # , 3.9]
+    runs-on: ubuntu-18.04  # Should match Dockerfile.olbase
```

Need to go one more 😅 
```suggestion
    runs-on: ubuntu-16.04  # Should match Dockerfile.olbase
```